nixos/modules/services/security/intune.nix

```diff
@@ -0,0 +1,51 @@
+{ config
+, pkgs
+, lib
+, ...
+}:
+with lib; let
+  cfg = config.services.intune;
+in
+{
+  options.services.intune = {
+    enable = mkEnableOption (lib.mdDoc "Microsoft Intune");
+  };
+
+
+  config = mkIf cfg.enable {
+    users.users.microsoft-identity-broker = {
+      group = "microsoft-identity-broker";
+      isSystemUser = true;
+    };
+
+    users.groups.microsoft-identity-broker = { };
+    environment.systemPackages = [ pkgs.microsoft-identity-broker pkgs.intune-portal ];
+    systemd.packages = [ pkgs.microsoft-identity-broker ];
+    systemd.services.microsoft-identity-device-broker.enable = true;
+    systemd.user.services.microsoft-identity-broker.enable = true;
+    systemd.tmpfiles.packages = [ pkgs.intune-portal ];
+    # Only really want the wants file set, but haven't been able to figure out how to do what without setting the whole thing here
```

Comment:
Comments go above the line they are about

Reply:
This was I think supposed to be a comment about the timer below, but I'm not sure at all what it says at this point. Removed the timer as the one that ships from upstream should just work.